vcpkg/TOC.yml

```diff
@@ -262,6 +262,8 @@
           href: commands/search.md
         - name: vcpkg update
           href: commands/update.md
+        - name: vcpkg x-add-version
+          href: commands/add-version.md

```

Comment:
We want to sort `x-` command as if the `x-` didn't exist. Given that, this should be after the Common Command Options in the ToC.